### PR TITLE
DescendantBooks: 'Report appearances' section option

### PR DESCRIPTION
**Pull request by giansalvo**
- added Report appearances section option

### CODE REVIEW

**Comment by prculley** [comment]
I think the option name "Include only once" and help description doesn't provide a good description of what the option does.  Maybe change popup to something like:
"Don't include the 'Report appearances' section with each person (which shows other places in the report where the person is mentioned)."

Or even better, "Include other person appearances" with popup: "Include the 'Report appearances' section with each person (which shows other places in the report where the person is mentioned)."  and invert your logic and have it checked by default.

File: DescendantBooks/DetailedDescendantBookReport.py [hunk 1]
```diff
@@ -442,11 +444,11 @@ def write_report(self):
             self.report_count += 1
             if len(self.ascendants) > 1:
                 self.title = \
-                    self._("%(report_count)s. Descendant Report for " \
+                    _("%(report_count)s. Descendant Report for " \
```
Comment (Nick-Hall) [inline DescendantBooks/DetailedDescendantBookReport.py]: This will break the deferred translation functionality.

File: DescendantBooks/DetailedDescendantBookReport.py [hunk 2]
```diff
@@ -531,7 +533,7 @@ def write_index_of_dates(self):
         This function prints the index of dates.
         """
         self.doc.start_paragraph("DDR-Title")
-        self.doc.write_text_citation("Index of Dates")
+        self.doc.write_text_citation(_("Index of Dates"))
```
Comment (Nick-Hall) [inline DescendantBooks/DetailedDescendantBookReport.py]: This should use deferred translation and belongs in a separate commit.

File: DescendantBooks/DetailedDescendantBookReport.py (hunk 1 above)
Comment (giansalvo) [reply DescendantBooks/DetailedDescendantBookReport.py]: Do you mean that I must revert all "\_(" to "self.\_(" or only here?

Comment (Nick-Hall) [reply DescendantBooks/DetailedDescendantBookReport.py]: Everywhere.  It's needed to run reports in different languages.

**Comment by prculley** [comment]
Made the aforementioned changes, built and uploaded.
